src/blueprint/operators/grpc.rs

```diff
@@ -129,20 +128,22 @@ impl TryFrom<String> for GrpcMethod {
     type Error = ValidationError<String>;
 
     fn try_from(value: String) -> Result<Self, Self::Error> {
-        let method: Vec<&str> = value.split('.').collect();
+        let method: Vec<&str> = value.rsplitn(3, '.').collect();
 
-        if method.len() != 3 {
-            return Err(ValidationError::new(format!(
-                "Invalid method format: {}. Expected format is <package/proto_id>.<service>.<method>",
+        match &method[..] {
+            &[name, service, id] => {
+                let method = GrpcMethod {
+                    id: id.to_owned(),
+                    service: format!("{id}.{service}"),
```

Comment:
Service should only have `service`.

Reply:
I did not edit any logic here.. it matches main branch.

Reply:
will have to change the implementation for that